pages/common/ngrep.md

```diff
@@ -0,0 +1,15 @@
+# ngrep
+
+> Ngrep is a pcap-aware tool that will allow you to specify extended regular or hexadecimal expressions to match against data payloads of packets.
+
+- Monitor all activity crossing source or destination port on network interface dev:
+
+`ngrep -d {{dev}} port {{port}}`
+
+- Monitor all activity crossing source or destination port on any network interface:
+
+`ngrep -d any port {{port}}`
+
+- Monitor all activity crossing source or destination port, show timestamp and format data in hex format:
+
+`ngrep -t -x -d any port {{port}}`
```

Comment:
Yes, split it in two examples is more clear. 
Thanks for your suggested comments, I will reorganize it all.